# HG changeset patch
# User Corey Thomasson <cthom.lists@gmail.com>
# Date 1366300796 14400
#      Thu Apr 18 11:59:56 2013 -0400
# Node ID 65546674d33666a4f73e638cc51a5da4b4bfc1ea
# Parent  5ec3f4aea91cfe1f854cd65ccaf73ccff1b64656
accumdim.m: fix bug and allow 6 arguments (bug #38770)

diff --git a/scripts/general/accumdim.m b/scripts/general/accumdim.m
--- a/scripts/general/accumdim.m
+++ b/scripts/general/accumdim.m
@@ -54,17 +54,17 @@
 ## @end group
 ## @end example
 ##
 ## @seealso{accumarray}
 ## @end deftypefn
 
 function A = accumdim (subs, vals, dim, n = 0, func = [], fillval = 0)
 
-  if (nargin < 2 || nargin > 5)
+  if (nargin < 2 || nargin > 6)
     print_usage ();
   endif
 
   if (isempty (fillval))
     fillval = 0;
   endif
 
   if (! isvector (subs))
